TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Constants.java

```diff
@@ -0,0 +1,44 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+
+import org.firstinspires.ftc.teamcode.util.MecanumDriveKinematics;
+
+public class Constants {
```

Comment:
Doesn't have to be now, but would it make sense to make these configurable instead of constant? Throughout FRC, it seemed that exposing all the config options was of value (though the approach of overriding the "constants" at runtime didn't feel like the right approach).